config/routes.config.js

```diff
@@ -0,0 +1,13 @@
+//creo el rouuter y lo exporto abajo de todo
+const express = require('express');
+const router = express.Router();
+
+const controller = require('../controllers/spotify.controller');
```

Comment:
```suggestion
const spotifyController = require('../controllers/spotify.controller');
```

Piensa que ahora mismo solo tienes un controlador pero esto es escalable, cada vez tendras mas controladores, por ello te recomiendo que empieces desde ya a ser descriptiva con el nombre de las variables asi de una pasada sabras que esa variable que es, esto tiene mucha importancia y las empresas lo tienen en cuenta (a un gran nivel de ellas vamos jejeje)